.ci/Jenkinsfile-extension-vsce

```diff
@@ -106,9 +106,17 @@ pipeline {
                             else
                                 ${WORKSPACE}/env/bin/pip install inmantals=="${LS_version}"
                             fi
-                            ls_version=$(${WORKSPACE}/env/bin/pip freeze | grep inmantals | cut -d'=' -f3
+                            ls_version=$(${WORKSPACE}/env/bin/pip freeze | grep inmantals | cut -d'=' -f3)
+
+                            if [[ -z "${ls_version}" ]]; then
+                                echo "Error: ls_version not found. Stopping execution."
+                                exit 1
+                            fi
+
                             echo "inmantals~=${ls_version}" > requirements.txt
-                            git commit -m "add requirements.txt file to tie the inmantals version (${ls_version}) to the extension" requirements.txt
+                            # force add requirements.txt as it is part of .gitignore
```

Comment:
The assumption here is that we don't have a `requirements.txt` file and we add one. If not we risk that we override the content of the existing requirements.txt file. Shouldn't we make this explicit and add a check that verifies that no requirements.txt exists.

Reply:
currently there should never be a requirements.txt. It is onlu used by the extention to install the right version of the LS. 
But I'm not oposed to check that to and to stop the pipeline if there is already one. I'll add the check